web/src/components/pages/PageFunds/index.js

```diff
@@ -126,7 +125,7 @@ function getGainInfo(state) {
         loss: cost > value
     });
 
-    return { classes, gainPct };
+    return map({ classes, gainPct });
```

Comment:
This is the most important change of the commit. Using a simple object resulted in the entire page component re-rendering whenever **any** reducer was run, because a new simple object (`gainInfo`) would fail an equality check despite having the same properties.
The solution is (of course) to make sure all the data is in the same form (immutable.js, in this case).